at_onboarding_flutter/lib/widgets/custom_dialog.dart

```diff
@@ -1143,6 +1147,21 @@ class _CustomDialogState extends State<CustomDialog> {
               cramSecret = await validatePerson(value, email, otp, context,
                   isConfirmation: true);
               return cramSecret;
+            } else if (value != null) {
+              AtStatusImpl atStatusImpl =
+                  AtStatusImpl(rootUrl: AppConstants.serverDomain);
+              AtStatus status = await atStatusImpl.get(value);
+
+              /// if atsign is not yet activated, it will be activated.
+              if (status.atSignStatus == AtSignStatus.unavailable &&
```

Comment:
use `status.atSignStatus.status()` to get correct status.

Reply:
Or else here is the fix https://github.com/atsign-foundation/at_libraries/pull/190